src/components/VaxLocation.tsx

```diff
@@ -21,15 +22,29 @@ import {
   Tooltip,
   Tr,
   useColorMode,
-  useColorModeValue as mode,
+  useColorModeValue,
   Wrap,
   WrapItem
 } from '@chakra-ui/react';
+import { DetailLokasi, VaccinationData } from 'data/types';
 import { formatDistanceToNow } from 'date-fns';
 import idLocale from 'date-fns/locale/id';
-import { ExternalLinkIcon } from "@chakra-ui/icons";
 
-export default function VaxLocation({ loading, location, isUserLocationExist }) {
+type DetailLokasiWithDistance = DetailLokasi & {
+  distance?: string | null;
+};
+
+export type VaccinationDataWithDistance = Omit<VaccinationData, 'detail_lokasi'> & {
+  detail_lokasi?: (DetailLokasiWithDistance | null)[] | null;
```

Comment:
btw can we refine this type? it feels too much

would be better if it's just `detail_lokasi?: DetailLokasiWithDistance[]`

Reply:
Added  in https://github.com/tibudiyanto/jakarta-vax-availability/pull/34/commits/6259a467af374b737231680a0d41c5ce906323d4